app/scripts/controllers/network/network.js

```diff
@@ -166,7 +166,8 @@ export default class NetworkController extends EventEmitter {
       return EIPS[1559];
     }
     const latestBlock = await this.getLatestBlock();
-    const supportsEIP1559 = latestBlock.baseFeePerGas !== undefined;
+    const supportsEIP1559 =
```

Comment:
Was this change because of an error that was occurin when on polygon?

Reply:
That is correct.